database/pipeline/table.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2022 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package pipeline
+
+import "github.com/go-vela/types/constants"
+
+const (
+	// CreatePostgresTable represents a query to create the Postgres pipelines table.
+	CreatePostgresTable = `
+CREATE TABLE
+IF NOT EXISTS
+pipelines (
+	id               SERIAL PRIMARY KEY,
+	repo_id          INTEGER,
+	number           INTEGER,
```

Comment:
Is this the build number?

Reply:
No, but it follows the same train of logic 👍 

Like `builds` today, each `pipeline` in the table will have a `number` field that increments based off the `repo_id`.

That field doesn't directly map to the build number because pipelines have a one-to-many relation with builds 😃 

You can have multiple builds all stemming from the same pipeline i.e. a build that gets restarted